elasticsearch/examples/migration/README.md

```diff
@@ -0,0 +1,86 @@
+* Check that your clusters health is green. If not abort and make sure your cluster is healthy before continuing.
+  ```
+  curl localhost:9200/_cluster/health
+  ```
+* Deploy new data nodes which will join the existing cluster. Take a look at the configuration in [data.yml](./data.yml)
+  ```
+  make data
+  ```
+* Check that the new nodes have joined the cluster (run this and any other curl commands from within one of your pods).
+  ```
+  curl localhost:9200/_cat/nodes
+  ```
+* Check that your cluster is still green. If so we can now start to scale down the existing data nodes. Assuming you have the default amount of data nodes (2) we now want to scale it down to 1.
+  ```
+  kubectl scale statefulsets my-release-elasticsearch-data --replicas=1
+  ```
+* Wait for your cluster to become green again
+  ```
+  watch 'curl -s localhost:9200/_cluster/health'
+  ```
+* Once the cluster is green we can scale down again.
+  ```
+  kubectl scale statefulsets my-release-elasticsearch-data --replicas=0
+  ```
+* Wait for the cluster to be green again.
+* OK. We now have all data nodes running in the new cluster. Time to replace the masters by firstly scaling down the masters from 3 to 2. Between each step make sure to wait for the cluster to become green again. During this process we will always make sure to keep at least 2 master nodes.
+  ```
+  kubectl scale statefulsets my-release-elasticsearch-master --replicas=2
+  ```
+* Now deploy a single new master so that we have 3 masters again. See [master.yml](./master.yml) for the configuration.
+  ```
+  make master
+  ```
+* Scale down old masters to 1
+  ```
+  kubectl scale statefulsets my-release-elasticsearch-master --replicas=1
+  ```
+* Edit the masters in [masters.yml](./masters.yml) to 2 and redeploy
+  ```
+  make master
+  ```
+* Scale down the old masters to 0
+  ```
+  kubectl scale statefulsets my-release-elasticsearch-master --replicas=0
+  ```
+* Edit the [masters.yml](./masters.yml) to have 3 replicas and remove the `discovery.zen.ping.unicast.hosts` entry from `extraEnvs` then redeploy the masters. This will make sure all 3 masters are running in the new cluster and are pointing at each other for discovery. 
+  ```
+  make master
+  ```
+* Remove the `discovery.zen.ping.unicast.hosts` entry from `extraEnvs` then redeploy the data nodes to make sure they are pointing at the new masters. 
+  ```
+  make data
+  ```
+* Deploy the client nodes
+  ```
+  make client
+  ```
+* Update any processes that are talking to the existing client nodes and point them to the new client nodes. Once this is done you can scale down the old client nodes
+  ```
+  kubectl scale deployment my-release-elasticsearch-client --replicas=0
+  ```
+* The migration should now be complete. After verifying that everything is working correctly you can cleanup and leftover resources from your old cluster.
```

Comment:
`s/you can cleanup and leftover/you can cleanup leftover/`?

Reply:
Fixed. Sorry English is not my first language, I grew up speaking Strayan.